src/parser/flow_ast.ml

```diff
@@ -73,6 +74,14 @@ and NumberLiteral : sig
   [@@deriving show]
 end = NumberLiteral
 
+and BigIntLiteral : sig
+  type t = {
+    value: float;
```

Comment:
Strictly speaking, I suppose we really should use some sort of OCaml bigint implementation here. A float might lose precision. But I think this is totally fine for now. Maybe in the future we can import Big_int from Core and use that :)

Reply:
Actually, would you mind renaming this field? Maybe something like 

```
approx_value: float; (* Warning! Might lose precision! *)
```

? AFAICT it looks like you're correctly using the `bigint` field everywhere, but anyone who uses the `value` field should be aware that it's approximate.